src/kube-runtime/src/runtime

```diff
@@ -46,19 +46,21 @@ PAI_RUNTIME_LOG_FILE=${PAI_LOG_DIR}/user.pai.all
 # priority=0
 source ${PAI_RUNTIME_DIR}/runtime_env.sh
 
-# setup ssh
-# priority=1
-${PAI_RUNTIME_DIR}/sshd.sh >> ${PAI_RUNTIME_LOG_FILE} 2>&1
-
 # check port conflict
 python ${PAI_RUNTIME_DIR}/port.py >> ${PAI_RUNTIME_LOG_FILE} 2>&1
 
+# execute preCommands generated by plugin
+${PAI_RUNTIME_DIR}/precommands.sh > >(tee -a ${PAI_LOG_DIR}/runtime.pai.stdout) 2> >(tee -a ${PAI_LOG_DIR}/runtime.pai.stderr >&2)
+
+# Put verbose output to user.pai.all, stdout to user.pai.stdout, stderr to user.pai.stderr
 # execute user commands
 # priority=100
 echo "[INFO] USER COMMAND START"
-# Put verbose output to runtime.log, stdout to user.pai.stdout, stderr to user.pai.stderr
 ${PAI_RUNTIME_DIR}/user.sh > >(tee -a ${PAI_LOG_DIR}/user.pai.stdout ${PAI_RUNTIME_LOG_FILE}) 2> >(tee -a ${PAI_LOG_DIR}/user.pai.stderr ${PAI_RUNTIME_LOG_FILE} >&2) &
```

Comment:
Change `PAI_RUNTIME_LOG_FILE` to another name? Or use `${PAI_RUNTIME_DIR}/user.pai.all` instead?

Reply:
Ok, since this is runtime user log and now we have runtme log. Will change name